client/indexer-service/Cargo.toml

```diff
@@ -0,0 +1,69 @@
+[package]
+name = "shc-indexer-service"
+version = "0.1.0"
+description = "Storage Hub Indexer Service."
+homepage = { workspace = true }
+license = { workspace = true }
+authors = { workspace = true }
+repository = { workspace = true }
+edition = { workspace = true }
+publish = false
+
+[lib]
+
+[lints]
+workspace = true
+
+[dependencies]
```

Comment:
It seems like there are a lot of unused dependencies. If that's the case, can we remove them?